pkg/config/config.go

```diff
@@ -78,8 +76,8 @@ func NewTarget(project string, regions []string, labelSelector string) *Target {
 
 // Validate checks if the configuration is valid.
 func (config *Config) Validate(cliMode bool) error {
-	if cliMode && config.Strategy.Interval <= 0 {
-		return errors.New("time interval must be greater than 0")
+	if cliMode && config.Strategy.HealthIntervalMinute <= 0 {
```

Comment:
it's a uint. :) how can be <= 0?
I recommend just making int

Reply:
Just of curiosity. If we're gonna check for `<= 0` we make it an `int`, why is an `int` better than `uint` in this type of contexts? Of course, this check should go away if we used `uint` as you point out.

Reply:
Mostly because we can validate ints better and we don't make advantage of uint.